testbed/tests/log_test.go

```diff
@@ -233,3 +240,114 @@ func TestLogOtlpSendingQueue(t *testing.T) {
 	})
 
 }
+
+func TestLogLargeFiles(t *testing.T) {
+	tests := []struct {
+		name         string
+		sender       testbed.DataSender
+		receiver     testbed.DataReceiver
+		loadOptions  testbed.LoadOptions
+		sleepSeconds int
+	}{
+		{
+			name:     "filelog-largefiles-2Gb-lifetime",
+			sender:   datasenders.NewFileLogWriter(),
+			receiver: testbed.NewOTLPDataReceiver(testutil.GetAvailablePort(t)),
+			loadOptions: testbed.LoadOptions{
+				DataItemsPerSecond: 200000,
+				ItemsPerBatch:      1,
+				Parallel:           100,
+			},
+			sleepSeconds: 100,
+		},
+		{
+			name:     "filelog-largefiles-6GB-lifetime",
+			sender:   datasenders.NewFileLogWriter(),
+			receiver: testbed.NewOTLPDataReceiver(testutil.GetAvailablePort(t)),
+			loadOptions: testbed.LoadOptions{
+				DataItemsPerSecond: 330000,
+				ItemsPerBatch:      10,
+				Parallel:           10,
+			},
+			sleepSeconds: 200,
+		},
+		{
+			name:     "filelog-largefiles-50MB/sec",
```

Comment:
What is the data rate of the other tests? e.g. `6GB-lifetime` appears to have nearly the same. Do we really need to run a separate test for this?

Reply:
I think I overlooked this test case. It was redundant, so I’ve removed the scenario.